Libraries/dnYara.Interop/dnYara.Interop.csproj

```diff
@@ -9,6 +9,7 @@
     <Company>Airbus CERT</Company>
     <Authors>Sylvain B. for Airbus CERT</Authors>
     <Copyright>Airbus CERT</Copyright>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
We must review structs to avoid "unsafe" until it's not necessary.

Reply:
I was able to remove all usage of unsafe thanks to the filling-in of the YR_MUTEX field in the YR_RULES struct.